CONTRIBUTING.md

```diff
@@ -137,7 +137,8 @@ gcloud services enable container.googleapis.com
 gcloud container clusters create demo --zone=us-central1-a \
     --machine-type=n1-standard-2 \
     --num-nodes=2 \
-    --enable-stackdriver-kubernetes \
+    --logging \
+    --monitoring \
```

Comment:
It looks like this command isn't valid. `--logging` and `--monitoring` [expect a `COMPONENT` field](https://cloud.google.com/sdk/gcloud/reference/container/clusters/create#--monitoring) (either `SYSTEM`, `WORKLOAD`, or `NONE`).

I'm not too familiar with these options, but from a skim it seems like the default of `SYSTEM` might be enough? So maybe we don't need these two lines?

What ever we end up using, we should make the [same change to the Makefile](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/blob/c984ea54dfd18a009d5018c66ea6e5a5b76f88b1/Makefile#L29)